src/openzaak/components/catalogi/migrations/0008_auto_20210203_0947.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2020 Dimpact
```

Comment:
@sergei-maertens should this be 2021?

Reply:
for new files, yes, the year that they've been added.